.github/workflows/maven-ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     permissions:
       contents: read
       packages: write
-    if: github.ref == 'refs/heads/develop'
+    #if: github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/groupStrat'
```

Comment:
Is the condition not required?

Reply:
Seems like we added refs/heads/groupStrat to force a publish to Maven repo for the branch

the original regs/heads/develop caused a merge to develop to publish

We can install locally for testing... but refs/heads/develop should be left intact and not commented out

Reply:
Updated in new push